I.write-tests/05-get-second-third-smallest/get-second-third.js

```diff
@@ -1,7 +1,7 @@
-module.exports = function (array) {
-    const newArray = array.slice()
-    newArray.sort(function (x, y) {
-        return x > y;
-    });
-    return [newArray[1], newArray[2]];
-};
\ No newline at end of file
+module.exports = function(array) {
+  const newArray = array.slice();
+  newArray.sort(function(x, y) {
```

Comment:
Hi @lpalma  thanks for reviewing  my code. I really tried and test it again it pass if you have used different data can you please send it to me, because it's already passed 👍